include/Model.hpp

```diff
@@ -16,44 +16,37 @@
 
 namespace indie {
 
-    /**struct Model;
-     * @brief Model 3D object
-     */  
+    /// @brief Model 3D object
     class Model {
         public:
             /**
              * @brief Construct a new model object
-             * @param filename path to the model file 
-             */
-            Model(const char *filename);
-            /**
-             * @brief destroy the model object
+             * @param filename path to the model file
+             * @param textureFile path to the texture file
              */
+            Model(const std::string &modelFile, const std::string &textureFile);
+            ///@brief destroy the model object
             ~Model();
             /**
              * @brief Draw the model 
              * @param position vector 3d to draw the model
-             * @param scale float of scaling (1.0f = default)
-             * @param tint Color of the model (WHITE = default)
-             */
-            void draw(Vector3 position, float scale, Color tint);
-            /**
-             * @brief unload the model object
+             * @param tint Color of the model (WHITE = using texture loaded)
```

Comment:
I think the documentation of this value is better like this because when you use WHITE as color you only use the texture loaded and when you use for exemple RED as color you will use the texture and a tint